wrangler-proto/src/main/java/io/cdap/wrangler/proto/workspace/v2/SampleSpec.java

```diff
@@ -56,6 +75,18 @@ public Set<StageSpec> getRelatedPlugins() {
     return relatedPlugins;
   }
 
+  public Set<SampleType> getSupportedSampleTypes() {
+    return supportedSampleTypes;
+  }
+
+  public SampleRequest getSampleRequest() {
+    return sampleRequest;
+  }
+
+  public void setSampleRequest(SampleRequest sampleRequest) {
```

Comment:
don't have set method unless there is a special use case, make the field `final` and use constructor to pass in the value

Reply:
Done, thanks!